sys/net/gnrc/network_layer/ipv6/netif/gnrc_ipv6_netif.c

```diff
@@ -87,6 +87,17 @@ static ipv6_addr_t *_add_addr_to_entry(gnrc_ipv6_netif_t *entry, const ipv6_addr
     tmp_addr->prefix_len = prefix_len;
     tmp_addr->flags = flags;
 
+#ifdef MODULE_GNRC_SIXLOWPAN_ND
+    if (entry->flags & GNRC_IPV6_NETIF_FLAGS_SIXLOWPAN) {
+        ipv6_addr_t *router = gnrc_ndp_internal_default_router();
+        if (router != NULL) {
+            tmp_addr->flags |= GNRC_IPV6_NETIF_ADDR_FLAGS_TENTATIVE;
+            gnrc_ndp_internal_send_nbr_sol(entry->pid, &tmp_addr->addr, router);
```

Comment:
and `s/gnrc_ndp_internal_send_nbr_sol(entry->pid, &tmp_addr->addr, router)/gnrc_ndp_internal_send_nbr_sol(entry->pid, &tmp_addr->addr, router, router)`